src/conn/conn_cache_pool.c

```diff
@@ -472,9 +472,9 @@ __cache_pool_assess(WT_SESSION_IMPL *session, uint64_t *phighest)
          * Figure out a delta since the last time we did an assessment for each metric we are
          * tracking. Watch out for wrapping of values.
          *
-         * Count pages read, assuming pages are 4KB.
+         * Count pages read, virtual memory page size.
```

Comment:
This comment should be updated.

Reply:
fixed https://github.com/wiredtiger/wiredtiger/pull/10030/commits/ee68a2d8ad4b194f222017ed49cdaf76f99124db